Sources/LiveViewNative/Views/Text Input and Output/TextFieldProtocol.swift

```diff
@@ -0,0 +1,216 @@
+            Task {
+                try await context.coordinator.pushEvent(type: "focus", event: event, value: element.buildPhxValuePayload())
+            }
+        } else {
+            guard let event = element.attributeValue(for: "phx-blur") else { return }
+            Task {
+                try await context.coordinator.pushEvent(type: "focus", event: event, value: element.buildPhxValuePayload())
+            }
+        }
+    }
+    
+    var placeholder: String? {
+        element.attributeValue(for: "placeholder")
+    }
+    
+    var textFieldStyle: PhxTextFieldStyle {
+        if let s = element.attributeValue(for: "text-field-style"),
+           let style = PhxTextFieldStyle(rawValue: s) {
+            return style
+        } else {
+            return .automatic
+        }
+    }
+    
+    var disableAutocorrection: Bool? {
+        switch element.attributeValue(for: "autocorrection") {
+        case "no":
+            return true
+        case "yes":
+            return false
+        default:
+            return nil
+        }
+    }
+    
+    var autocapitalization: TextInputAutocapitalization? {
+        switch element.attributeValue(for: "autocapitalization") {
+        case "sentences":
+            return .sentences
+        case "words":
+            return .words
+        case "characters":
+            return .characters
+        case "never":
+            return .never
+        default:
+            return nil
+        }
+    }
+    
+    var keyboard: UIKeyboardType? {
+        switch element.attributeValue(for: "keyboard") {
+        case "ascii-capable":
+            return .asciiCapable
+        case "numbers-and-punctuation":
+            return .numbersAndPunctuation
+        case "url":
+            return .URL
+        case "number-pad":
+            return .numberPad
+        case "phone-pad":
+            return .phonePad
+        case "name-phone-pad":
+            return .namePhonePad
+        case "email-address":
+            return .emailAddress
+        case "decimal-pad":
+            return .decimalPad
+        case "twitter":
+            return .twitter
+        case "web-search":
+            return .webSearch
+        case "ascii-capable-number-pad":
+            return .asciiCapableNumberPad
+        default:
+            return nil
+        }
+    }
+    
+    var submitLabel: SubmitLabel? {
+        switch element.attributeValue(for: "submit-label") {
+        case "done":
+            return .done
+        case "go":
+            return .go
+        case "send":
+            return .send
+        case "join":
+            return .join
+        case "route":
+            return .route
+        case "search":
+            return .search
+        case "return":
+            return .`return`
+        case "next":
+            return .next
+        case "continue":
+            return .continue
+        default:
+            return nil
+        }
+    }
+}
+
+enum PhxTextFieldStyle: String {
```

Comment:
We're moving away from the Phx prefix, can you rename this and the various View extensions?

Reply:
I removed the prefix from the enum, as well as the `PhxButtonStyle` enum. For the `View` extensions, I used the prefix `apply`, such as `applyTextFieldStyle` because of the name collision with the built-in modifier.